services/motion/explore/explore.go

```diff
@@ -332,7 +343,16 @@ func (ms *explore) checkForObstacles(
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-			currentPose := spatialmath.NewZeroPose()
+
+			currentInputs, err := kb.CurrentInputs(ctx)
```

Comment:
[note] Update explore to use the currentInputs and calculate the currentPose of the kinematic base instead of assuming (0,0,0)